src/styles/blocks/navigation.scss

```diff
@@ -0,0 +1,45 @@
+.navigation {
+  &__list {
+    margin: 0;
+    padding: 0;
+    display: flex;
+    list-style-type: none;
+    align-items: end;
+  }
+
+  &__item {
+    text-transform: uppercase;
+  }
+
+  &__item:not(:last-child) {
+    margin-right: 20px;
+  }
+
+  &__link {
+    display: block;
+    height: 60px;
+    line-height: 60px;
+    text-decoration: none;
+    color: $nav-link-color;
+
+    &:hover {
+      color: $nav-link-hover-color;
```

Comment:
Variables should be defined using kebab-case according to the [Code Style Rules ❗️](https://mate-academy.github.io/layout_task-guideline/html-css-code-style-rules). The variable '$nav-link-hover-color' should be renamed to '$nav-link-hover-color'.